Extensions/AdMob/JsExtension.js

```diff
@@ -302,6 +302,15 @@ module.exports = {
         '',
         false
       )
+      .addParameter(
+        'yesorno',
+        _('Offset? (outside webview otherwise)'),
+        '',
+        false
+      )
+      .setParameterLongDescription(
+        'By setting offset to true, just overlapping the banner on top of the content.\nkeep false for the default setting where the banner pushes the webview, this setting creates a black space around the banner, and can also create borders around your game.'
```

Comment:
This sentence should be translated. You can do so by using `_(...)`  as done for the parameter label just above

Reply:
@devgiordane Can you remove this entirely?

Reply:
Hello, I tried to remove this change, and keep only the offset commit, I have little familiarity with git, and I don't know if it worked.

Reply:
It did not but I used GitHub to directly edit the file and remove the changes :)